### PR TITLE
feat(projects): add a flag to enable/disable displaying project's vulnerabilities

### PR DESCRIPTION
**Pull request by alexbrdn**
some users do not want to reveal vuls to their project.

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf [hunk 1]
```diff
@@ -33,11 +33,19 @@
...
-<p>Security Vulnerability Monitoring:
+<p>
+    <span>Security Vulnerability Monitoring:
         <span
             <core_rt:if test="${project.enableSvm}">class="notificationBulletInline backgroundOK">enabled</core_rt:if>
             <core_rt:if test="${not project.enableSvm}">class="notificationBulletInline backgroundGrey">disabled</core_rt:if>
         </span>
+    </span>
+    <span>Security Vulnerabilities Display:
+        <span
```
Comment (kallesoranko) [inline frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/vulnerabilities.jspf]: Missing "bigger than" character?

**Comment by maxhbr** [comment]
I am not sure whether on the details page of a vulnerability the affected projects are listed. In this case the projects, which do not want vulnerability tracking, should be removed from there.

**Comment by alexbrdn** [comment]
No, the projects are not listed on the vulnerability details page.

**Comment by maierthomas** [comment]
If we hide the vulnerabilities should we also hide these information?
image
image

**Comment by adrian-evo** [comment]
Tested and the flag works fine, except the mention from @maierthomas above.

**Comment by alexbrdn** [comment]
I hid the vulnerabilities counts. Please retest

**Comment by maierthomas** [comment]
tested successfully
Disabled: Vulnerabilites ? | vulnerabilities out of ? in total. 
Enabled: Vulnerabilites 0/0 | vulnerabilities out of 0 in total. 

image

**Pull request merged**